src/dataServices/dynamoDbBundleService.test.ts

```diff
@@ -377,8 +385,26 @@ describe('atomicallyReadWriteResources', () => {
                 ],
             });
 
-            // 2. change Patient record's documentStatus to be 'AVAILABLE'
+            // 2. Update ttlInSeconds for previous version
             expect(transactWriteItemSpy.getCall(2).args[0]).toStrictEqual({
+                TransactItems: [
+                    {
+                        Update: {
+                            TableName: '',
+                            Key: { id: { S: id }, vid: { N: oldVid.toString() } },
+                            UpdateExpression: 'set ttlInSeconds = :ttlInSeconds',

```

Comment:
We could just change the constant here in persistence and the resource table's ttl configuration. I'd prefer to not rename across all the code though because I've already done that 1x already. :-)

Reply:
Updated the field name to _ttlInSeconds to match _resources convention. I should have caught this earlier. Also, so someone else doesn't hit this problem, dynamoDB's update expression doesn't allow field names with `_` characters. You have to use the ExpressionAttributeNames to map the field name per https://github.com/aws/aws-sdk-js/issues/1691